### PR TITLE
Feature/about page

### PR DESCRIPTION
**Pull request by ylberprapashtica**
I created an About Us page to show information about the team.

- [ ] Check for typos
- [ ] Is the clean
- [ ] Can this be optimize

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Page</title>
+</head>
+<body>
+    <h1>About page</h1>
```
Comment (ylberprapashtica) [inline about.html:9]: You wrote page with lowercase, make the P uppercase.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        Commodi vitae sit ut temporibus ea eaque at corrupti qui accusantium, cumque, unde officiis reprehenderit perferendis debitis itaque nulla quo optio et nam quos? Earum ut voluptatem consequatur libero aspernatur!
+        Consequuntur dignissimos dolorem officia quidem fugiat molestias ut, alias laborum cumque, sunt omnis ipsam voluptas amet sed cum enim odit quos sint? Officiis aliquam cupiditate quibusdam in vel consectetur perferendis.
+        Corporis neque veniam obcaecati quam excepturi officia cupiditate earum recusandae, consectetur nam! Deleniti corporis distinctio recusandae cupiditate. Quae quas illum expedita? Quidem, eveniet eaque natus iusto tempore possimus dolorem. Delectus.
+        Facilis a nostrum doloremque nihil eligendi enim sunt, maxime debitis cupiditate, aliquam, repudiandae quidem ullam sed dolorum quas explicabo dolorem similique facere consequatur quia fuga. Beatae natus sed deserunt accusantium!
+        Animi minus magni, nam repellat unde nemo aut! Hic iure totam maxime? Sint dolores sunt veritatis libero quae vero? Doloribus quos officia voluptates doloremque eum corrupti recusandae ratione cum maiores.
```
Comment (ylberprapashtica) [inline about.html:16]: What did you want to do with this part?

**Review by ylberprapashtica — COMMENTED** [review]
I just have some commets

File: about.html (hunk 1 above)
Comment (RedonTahiri) [inline about.html:9]: Make it h2

**Review by RedonTahiri — CHANGES_REQUESTED** [review]
Looks good to me, but you need to make this minor change.

File: about.html (hunk 1 above)
Comment (ylberprapashtica) [reply about.html:9]: But this is the title of the page, it should be h1

File: sub-about.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sub about Page</title>
```
Comment (RedonTahiri) [inline sub-about.html:6]: Change the title to blablabla

**Review by RedonTahiri — CHANGES_REQUESTED** [review]

File: sub-about.html (hunk 3 above)
Comment (ylberprapashtica) [reply sub-about.html]: I made the change

Comment (RedonTahiri) [reply sub-about.html]: It's okay now.

**Pull request merged**